main_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"net"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUsage(t *testing.T) {
```

Comment:
Testing the usage message. That's hardcore.

Reply:
You have no idea what hardcore is it seems.